src/components/Popups/index.tsx

```diff
@@ -37,24 +39,36 @@ const StopOverflowQuery = `@media screen and (min-width: ${MEDIA_WIDTHS.deprecat
   MEDIA_WIDTHS.deprecated_upToMedium + 500
 }px)`
 
-const FixedPopupColumn = styled(AutoColumn)<{ extraPadding: boolean; xlPadding: boolean }>`
+type TopDistanceModifiers = {
+  bannerVisible: boolean
+  drawerOpen: boolean
+}
+
+const getTopDistance = ({ drawerOpen, bannerVisible }: TopDistanceModifiers) => {
+  return 64 + (drawerOpen ? -50 : 0) + (bannerVisible ? 8 : 0)
```

Comment:
any chance we can reference variable here and share it with the styled component in question? if possible it would be nice so we don't have to update in 2 places in the future